x-pack/platform/plugins/shared/stack_connectors/public/connector_types/email/email_connector.tsx

```diff
@@ -159,29 +181,52 @@ export const EmailActionConnectorFields: React.FunctionComponent<ActionConnector
       </EuiFlexGroup>
       <EuiFlexGroup justifyContent="spaceBetween">
         <EuiFlexItem>
-          <UseField
-            path="config.service"
-            component={SelectField}
-            config={{
-              label: i18n.SERVICE_LABEL,
-              validations: [
-                {
-                  validator: emptyField(i18n.SERVICE_REQUIRED),
+          {availableEmailServices.length === 1 ? (
```

Comment:
I think it is better if we do not introduce a separate field just for when the available services are. You could disable the selection (`disabled: readOnly || isLoading || availableEmailServices.length <= 1,`). I tested, and the result is visually the same. Nevertheless, I would say not to disable at all because users may want to change an already configured connector to the new valid service. Otherwise, they would have to delete that one and create a new one, meaning they would have to update all rules to use the new connector.